packages/augur-ui/src/modules/create-market/constants.ts

```diff
@@ -509,6 +510,10 @@ export const VALIDATION_ATTRIBUTES = {
 };
 
 export const TemplateBannerText = {
-  [INDEXES] : 'Enter the full name of the index to ensure the market resolves as valid, i.e. S & P 500 Index',
-  [STOCKS] : 'Enter stock ticker symbol to ensure the market resolves as valid, i.e. AAPL'
-}
+  indexes:
```

Comment:
why do you not use the constant anymore?

Reply:
needed lower case for comparison. 
couldn't easily compare user input to constant (cap case) value

Reply:
constants are cap case, just used lowcase key instead of creating a helper method to cap case user inputted string to get message.